### ISSUE TITLE
Create a PDF for the Translation Notes for a Book

### ISSUE THREAD

**Issue by mandolyte** [unfoldingWord/gateway-admin#166]
Here is the original instruction: 

> Write up a gatewayAdmin Epic for creating and integrating a GL quote OCP Open Component Package. @mandolyte @kintsoogi  

At a high level, here are the steps with dependencies (predecessors) marked with square brackets.

1. Identify and extract the fundamental logic for calculating the GL Quote from data in a row in a TN in TSV7 format
2. Create a package for this with documentation, Styleguidist examples, and publish to NPM [1]
3. In gA, design a UI that allows the user to create and show the Translation Notes for a file in a separate window, which then can be printed or saved as a PDF [2]
4. In gA, implement the aforementioned design [3]

@birchamp @kintsoogi Above are the four issues that I can think of that should be in this epic. Thoughts?

**Comment by birchamp** [comment]
@mandolyte this looks right to me.  Probably @richmahn should be in on this conversation as well as he's done a lot of thinking along this line.  You could write up an issue on #1 @mandolyte based on what you've discovered looking at TWLs in gE and your notes on Discord.

**Comment by PhotoNomad0** [comment]
See SPIKE: https://github.com/unfoldingWord/gateway-edit/issues/357

**Comment by richmahn** [comment]
@birchamp @mandolyte Yeah, was wondering in #1 if we have what we need, both data and logic, for figuring out the GL quote for a TN. TN GL quote finding can be quite complex, especially when it comes to those with two or more separate parts (divided by ellipse), which I've found don't get accurately covered by a single occurrence number.

Yet I assume if translateion-helps-rcl has it figured out, then that should be good.
